lib/core/base/audit.js

```diff
@@ -40,8 +40,191 @@ function Audit(audit) {
 
 	this.defaultConfig = audit;
 	this._init();
+
+	// A copy of the "default" locale. This will be set if the user
+	// provides a new locale to `axe.configure()` and used to undo
+	// changes in `axe.reset()`.
+	this._previousLocale = null;
 }
 
+/**
+ * Build and set the previous locale. Will noop if a previous
+ * locale was already set, as we want the ability to "reset"
+ * to the default ("first") configuration.
+ */
+
+Audit.prototype._setPreviousLocale = function() {
+	if (this._previousLocale) {
+		return;
+	}
+
+	const locale = {
+		checks: {},
```

Comment:
Should we keep track of a `lang` key here for any reason? The existing locale files include that alongside rules and checks.

Reply:
It's not used by anything, so saving it didn't seem super important. What do you think we need it for?

Reply:
It's a good point. I think we should include a lang key on the results object. I think we should do that separately from this PR though.